src/ResourceManager/Cdn/Commands.Cdn.Test/Commands.Cdn.Test.csproj

```diff
@@ -78,6 +77,14 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.2.28.3\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@prakharsharma10 you can remove the references to `Microsoft.Rest.ClientRuntime` and Microsoft.Rest.ClientRuntime.Azure` in this file since they are pulled in from a separate targets file

Reply:
@cormacpayne We are using clientruntime version 2.3.10 instead of 2.3.11 Therefore, we need to have these references.

Reply:
@prakharsharma10 the SDK has a minimum version dependency on 2.3.10, so it is able to use 2.3.11. Deleting the added reference will use 2.3.11 (and 3.3.12 of the other package), which is what we want